### ISSUE TITLE
About hyper-parameters

### ISSUE THREAD

**Issue by jelleopard** [choyingw/SynergyNet#8]
It seems that some hyper-parameter settings in the code (train_script.sh) are inconsistent with those in the paper, for example, learning rate (0.027 vs. 0.08), loss weight \lambda_{2} (0.05 vs. 0.03), batch size (900 vs. 1024), milestones (48 & 64 vs. 30 & 40), epoch number (50 vs. 80) and lr decay (0.2 vs. 0.1). 
Of course these numbers are adjustable but important in the experiments. I want to know how to set these hyper-parameters with backbone mobilenetv2 to get a good performance as yours.

**Comment by choyingw** [comment]
The checkpoint we released  is trained by the setting described in the paper. The default hyper-parameter in the code is something we tried recently fo explore more parameter tuning. I'll change the default back to those in the paper.

**Comment by jelleopard** [comment]
Thank you for your response. I have no problems, so I will close this issue.
